ui/cypress/e2e/explorer.test.ts

```diff
@@ -43,6 +43,36 @@ describe('DataExplorer', () => {
     })
   })
 
+  describe('data-explorer state', () => {
+
+    it('should persist and display last submitted script editor script ', () => {
+      const fluxCode = 'from(bucket: "_monitoring")'
+      cy.getByTestID('switch-to-script-editor').click()
+      cy.get('.flux-editor').within(() => {
+        cy.get('.view-lines').type(fluxCode)
+      })
+      cy.contains('Submit').click()
+      cy.getByTestID('nav-item-tasks').click()
+      cy.getByTestID('nav-item-data-explorer').click()
+
+      cy.get('.flux-editor').within(() => {
+        cy.get('.view-lines').contains(fluxCode).should('exist')
```

Comment:
you can safely drop the `should('exist')` if it follows a `contains` function.  https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Default-Assertions


would 
```ts
  cy.contains(fluxCode)
```
work? or does this need to be scoped to `.view-lines`?

if you can just find `fluxCode` anywhere, this can be simplified to

```suggestion
        cy..contains(fluxCode)
```